Procfile

```diff
@@ -1,2 +1,3 @@
 web: bin/upgrade && PYTHONUNBUFFERED=1 bin/web --no-debug --addr=:${PORT}
 worker: bin/upgrade && PYTHONUNBUFFERED=1 bin/worker --no-debug -n 4 -l ${LOG_LEVEL}
+static: node_modules/.bin/webpack -d --watch
```

Comment:
This shouldn't be a part of the `Procfile`, it should only be in `Procfile.dev` since it's not used for production.